app.js

```diff
@@ -24,13 +21,12 @@ app.get('/api/v1/projects', async (request, response) => {
     }
 });
 
-//Get Sepcific Project
 app.get('/api/v1/projects/:id', async (request, response) => {
   const {id} = request.params
   try{
     const project = await database('projects').where('id', id).select()
     if(project.length) return response.status(200).json(project)
-    if(!project.length) return response.status(404).json({error: `No project found with id of ${id}`})
+    if(!project.length) return response.status(404).json(`{Error: No palette found with ${id}}`)
```

Comment:
This line is not the most recent change.

Reply:
if(!project.length) return response.status(404).json({error: `No project found with id of ${id}`})